api/client.yaml

```diff
@@ -35,6 +35,57 @@ paths:
       responses:
         '200':
           description: Service is running properly
+  /configuration/transfers:
```

Comment:
nit:
Shortening to `/config/transfers` could be nice.

Reply:
Agreed, it would. I just followed the convention already in use by customers. https://github.com/moov-io/customers/blob/master/api/client.yaml#L38